tests/ui_tools/test_boxes.py

```diff
@@ -41,8 +42,50 @@ def test_init(self, write_box):
     ])
     def test_generic_autocomplete_no_prefix(self, mocker, write_box, text,
                                             state):
+        write_box.view.set_typeahead_footer = mocker.patch(
+                                'zulipterminal.ui.View.set_typeahead_footer')
```

Comment:
If this patch works, this shouldn't need such a complex assignment, as per the other test in this commit.

Reply:
What do you mean by this? Complex assignment of the patch?

Reply:
As per the other test, you have eg. `set_typeahead_footer = mocker.patch(...`, which is enough to patch the method, and then you can test that local variable later.

Reply:
I have the same assignment in the other test too. I tried this and a local assignment does not work. Probably because `set_typeahead_footer` is called using `write_box.view`?

Reply:
Ah, right, this doesn't need to be patched at all.